src/data/rel.lean

```diff
@@ -88,6 +94,7 @@ end
 
 lemma image_univ : r.image set.univ = r.codom := by { ext y, simp [mem_image, codom] }
 
+/-- Preimage of a set under a relation `r`. Same as the image of `s` under `r.flip` -/
```

Comment:
```suggestion
/-- Preimage of a set under a relation `r`. Same as the image of `s` under `r.inv` -/
```
It seems like we don't have the lemma saying that `r.preimage s = r.inv.image s` or vice versa. Even if you meant this to be a docs-only PR, I guess it would be nice to have it.

Reply:
I'd prefer not to touch code in this PR. I'll add this lemma to the next PR.